rest/class-boldgrid-backup-rest-test.php

```diff
@@ -0,0 +1,107 @@
+<?php
+/**
+ * File: class-boldgrid-backup-rest-test.php
+ *
+ * @link       https://www.boldgrid.com
+ * @since      X.X.X
+ *
+ * @package    Boldgrid_Backup
+ * @copyright  BoldGrid
+ * @version    $Id$
+ * @author     BoldGrid <support@boldgrid.com>
+ */
+
+/**
+ * Class: Boldgrid_Backup_Rest_Test
+ *
+ * REST endpoints to access the backup preflight check results.
+ *
+ * @since X.X.X
+ */
+class Boldgrid_Backup_Rest_Test extends Boldgrid_Backup_Rest_Controller {
+
+	/**
+	 * Resource name.
+	 *
+	 * @since  X.X.X
+	 * @access private
+	 * @var    string
+	 */
+	protected $resource = 'test';
+
+	/**
+	 * Register the routes for the objects of the controller.
+	 *
+	 * @since X.X.X
+	 */
+	public function register_routes() {
+		$this->register_get();
+	}
+
+	/**
+	 * Register the route for getting test results.
+	 *
+	 * @since X.X.X
+	 */
+	public function register_get() {
+		register_rest_route( $this->namespace, '/' . $this->resource, [
+			[
+				'methods'             => WP_REST_Server::READABLE,
+				'callback'            => [ $this, 'get_item' ],
+				'permission_callback' => [ $this, 'permission_check' ],
+			],
+			'schema' => [ $this, 'get_schema' ],
+		] );
+	}
+
+	/**
+	 * Prepare the item for the REST response.
+	 *
+	 * @since X.X.X
+	 *
+	 * @param mixed $item WordPress representation of the item.
+	 * @param WP_REST_Request $request Request object.
+	 * @return mixed
+	 */
+	public function prepare_item_for_response( $item, $request ) {
+		return $this->filter_schema_properties( $item );
+	}
+
+	/**
+	 * Get schema for results.
+	 *
+	 * @since X.X.X
+	 *
+	 * @return array Schema Format.
+	 */
+	public function get_schema() {
+		$schema = [
```

Comment:
Arrays should be declared with long array syntax per WordPress coding standards
https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#declaring-arrays

Reply:
See similar comment above.